nebullvm/optimizers/tensor_rt.py

```diff
@@ -47,6 +51,25 @@
                 "No TensorRT valid installation has been found. "
                 "It won't be possible to use it in the following."
             )
+    try:
+        import torch_tensorrt
+    except ImportError:
+        # TODO: Remove the False flag for allowing Torch-TensorRT to
+        #  be installed by the Auto-Installer.
+        if False and not NO_COMPILER_INSTALLATION:
```

Comment:
I think we can activate this installation by default. Just check if cuda is available.